service-spec/schema-validator/src/main/resources/hspec/0.6.1.1/service-spec.json

```diff
@@ -83,23 +86,15 @@
       "type": "object"
     },
     "io.hyscale.api.image.Image": {
-      "description": "Describes container Image, could be from dockerfile or buildSpec",
+      "description": "Describes container Image could be from dockerfile or buildSpec",
```

Comment:
Describes container image and its source, whether the image build from binaries or dockerfile with buildSpec inside image.